erts/doc/src/erlang.xml

```diff
@@ -1618,28 +1698,58 @@ b</pre>
       <name name="error" arity="2" since=""/>
       <fsummary>Stop execution with a specified reason.</fsummary>
       <desc>
-        <p>Stops the execution of the calling process with the reason
-          <c><anno>Reason</anno></c>, where <c><anno>Reason</anno></c>
+        <p>Raises exception of class <c>error</c> with the reason
+        <c><anno>Reason</anno></c>, where <c><anno>Reason</anno></c>
           is any term. The exit reason is
           <c>{<anno>Reason</anno>, Where}</c>, where <c>Where</c>
           is a list of the functions most recently called (the current
           function first). <c><anno>Args</anno></c> is expected to be the
           list of arguments for the current function; in Beam it is used
           to provide the arguments for the current function in
           the term <c>Where</c>. As evaluating this function causes
-          the process to terminate, it has no return value.</p>
+          an exception to be thrown, it has no return value.
```

Comment:
I think it is better to avoid "thrown" as it could be misunderstood.

```suggestion
          an exception to be raised, it has no return value.
```